codespell_lib/_codespell.py

```diff
@@ -280,7 +285,7 @@ def parse_options(args):
                              'patterns to ignore by treating as whitespace. '
                              'When writing regexes, consider ensuring there '
                              'are boundary non-word chars, e.g., '
-                             '"\\Wmatch\\W". Defaults to empty/disabled.')
+                             '"\\bmatch\\b". Defaults to empty/disabled.')
```

Comment:
Excellent spot!

I don't know if you fancy fixing the same error here:
https://github.com/codespell-project/codespell/blob/5a8085ac47e4aeeb3a8983c64355a152ff00ad4c/codespell_lib/tests/test_basic.py#L487-L488

Edit, you have already thanks!